manifests/init.pp

```diff
@@ -104,6 +111,9 @@
     }
   }
 
+  $caddy_dl_url = "${caddy_url}/${arch}?plugins=${caddy_features}&license=${caddy_license}&telemetry=${caddy_telemetry}"
+  $caddy_dl_dir = "${caddy_tmp_dir}/caddy_linux_${$arch}_custom.tar.gz"
+
   group { $caddy_group:
```

Comment:
Managing users and groups should always be optional.

Reply:
I disagree. We should use a dedicated user to run Caddy. We should implement the  parameters ```manage_user``` and ```manage_group```, because Caddy 2 will provide official packages for ```.deb``` and ```.rpm```.

Reply:
manage_user and manage_group are what I had in mind, in order to make it optional :) Agree it should use its own user, though sometimes that's done through LDAP instead of local user/group.